### PR TITLE
Correcting qty units prescription in other drugs dispensed and changing autocomplete of other drugs from answerConceptSet to answerClasses

### PR DESCRIPTION
**Pull request by slubwama**
Correcting qty units prescription in other drugs dispensed. 
https://app.asana.com/0/1133167201254913/1163458026660473

Changing autocomplete of other drugs from answerConceptSet to answerClasses
https://app.asana.com/0/1133167201254913/1163458026660474

This PR requires https://github.com/METS-Programme/openmrs-module-aijar/pull/470 to be merged first

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 1]
```diff
@@ -1535,40 +1535,32 @@
...
-                                                                        <div class="col-md-2">
+                                                                        <div class="col-2">
                                                                             <br/>
-                                                                            <obs conceptId="1732"/>
+                                                                            <obs conceptId="1732" labelText="Duration unit"/>
                                                                         </div>
-                                                                        <div class="col-md-2">
-                                                                            <div>
+                                                                        <div class="col-2">
+                                                                            <div style="font-size: 25px;margin-top: 26px;">
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml]: @slubwama use Boostrap styles not hardcoded parameters

File: omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml [hunk 2]
```diff
@@ -2446,7 +2438,7 @@
                                                         </div>
                                                         <div>
                                                             <label>CD4%</label>
-                                                            <obs id="viral-load-results-qualitative" conceptId="730"
+                                                            <obs conceptId="730"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:2444]: @slubwama Where are the numeric ID values for the actual viral load values when detected

Comment (slubwama) [reply omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml:2444]: @ssmusoke this was wrongly placed on the CD4 result set so Its being removed to catch potential wrong validations

**Comment by ssmusoke** [comment]
@slubwama How to I verify that this change makes sense - what should I be able to see on the UI

**Comment by slubwama** [comment]
@ssmusoke these two show old and new look of form
new-other-drugs
new look

other-meds-dispensed
Old look

**Comment by Mwanje** [comment]
@slubwama can't access the Clinical Assessment page see gist https://gist.github.com/Mwanje/9e76a8ad238648f89c364a0998287339

**Comment by slubwama** [comment]
> @slubwama can't access the Clinical Assessment page see gist https://gist.github.com/Mwanje/9e76a8ad238648f89c364a0998287339

@Mwanje you need to have aijar pulled from upstream. That concept being complained about 165791 
 was added.

**Pull request merged**